### PR TITLE
content updated

### PR DESCRIPTION
**Pull request by mariumfirdous**
### Description

What component does this PR affect? 

- [ ] Server
- [ ] Console
- [ ] CLI
- [x] Docs
- [ ] Community Content
- [ ] Build System

Requires changes from other components? If yes, please mark the components:

- [ ] Server
- [ ] Console
- [ ] CLI
- [x] Docs
- [ ] Community Content
- [ ] Build System

### Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Docs update
- [ ] Community content

### Checklist:

- [x] I have read the **[contributing guide](https://github.com/hasura/graphql-engine/blob/master/CONTRIBUTING.md)** and my code conforms to the guidelines.
- [ ] This change requires a change in the documentation. 
- [ ] I have updated the documentation accordingly.
- [ ] I have added required tests.

### CODE REVIEW

**Comment by mariumfirdous** [comment]
#754

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -6,9 +6,9 @@ worst that can happen is that you'll be politely asked to change something. We
 appreciate any sort of contributions and don't want a wall of rules to get in
 the way of that.
 
-However, for those individuals who want a bit more guidance on the best way to
-contribute to the project, read on. This document will cover what we're looking
-for. By addressing all the points we're looking for, it raises the chances we
+However, for those who want a bit more guidance on the best way to
```
Comment (shahidhk) [inline CONTRIBUTING.md]: This sentence is already re-factored in an earlier PR. Can you undo the changes here?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -56,10 +56,10 @@ before-mentioned sections).
 
 ## Common guidelines
 
-- Please make sure there is an issue associated with the work that you're doing.
+- Please make sure there is an issue associated with the work that you are doing.
 
-- If you're working on an issue, please comment that you are doing so to prevent
-  duplicate work by others also.
+- If you are working on an issue, please comment about the same to prevent
+  conflicts.
```
Comment (shahidhk) [inline CONTRIBUTING.md]: Commenting that you're working is to prevent duplication of work, not to avoid conflicts. :smile:

**Comment by dsandip** [comment]
@mariumfirdous Closing this one till the requested changes are pushed here.
